dbt-athena/src/dbt/include/athena/macros/materializations/models/incremental/incremental.sql

```diff
@@ -57,7 +70,9 @@
       {% endcall %}
     {%- endif -%}
     {% set build_sql = "select '" ~ query_result ~ "'" -%}
-  {% elif partitioned_by is not none and strategy == 'insert_overwrite' %}
+
+  -- Insert Overwrite and Microbatch Strategies --
+  {% elif strategy in ("insert_overwrite", "microbatch") %}
```

Comment:
Does this work with Iceberg? 
I thought Iceberg only was only officially supporting `append` and `merge` according to https://dbt-athena.github.io/docs/configuration/materializations/iceberg#incremental-tables

Reply:
It did not, but I've now updated the PR to also have microbatch support for iceberg tables via the merge strategy 🙂